.ci/lint

```diff
@@ -45,38 +46,38 @@ pipeline {
         }
         stage('linting chg') {
             parallel {
-                stage('lint salt') {
+                stage('lint salt chg') {
                     when {
                         expression { return readFile('file-list-changed.log') =~ /(?i)(^|\n)(salt\/.*\.py|setup\.py)\n/ }
                     }
                     steps {
                         sh '''
                         eval "$(pyenv init - --no-rehash)"
                         # tee makes the exit/return code always 0
-                        grep -Ei '^salt/.*\\.py$|^setup\\.py$' file-list-changed.log | (xargs -r '--delimiter=\\n' tox -e pylint-salt;echo "$?" >  pylint-salt-chg.exit) | tee pylint-report-salt.log
+                        grep -Ei '^salt/.*\\.py$|^setup\\.py$' file-list-changed.log | (xargs -r '--delimiter=\\n' tox -e pylint-salt ; echo "$?" >  pylint-salt-chg.exit) | tee pylint-report-salt-chg.log
                         # remove color escape coding
                         sed -ri 's/\\x1B\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g' pylint-report-salt.log
                         read rc_exit < pylint-salt-chg.exit
                         exit "$rc_exit"
                         '''
-                        archiveArtifacts artifacts: 'pylint-report-salt.log'
+                        archiveArtifacts artifacts: 'pylint-report-salt-chg.log'
                     }
                 }
-                stage('lint test') {
+                stage('lint test chg') {
                     when {
                         expression { return readFile('file-list-changed.log') =~ /(?i)(^|\n)tests\/.*\.py\n/ }
                     }
                     steps {
                         sh '''
                         eval "$(pyenv init - --no-rehash)"
                         # tee makes the exit/return code always 0
-                        grep -Ei '^tests/.*\\.py$' file-list-changed.log | (xargs -r '--delimiter=\\n' tox -e pylint-tests;echo "$?" > pylint-tests-chg.exit) | tee pylint-report-tests.log
+                        grep -Ei '^tests/.*\\.py$' file-list-changed.log | (xargs -r '--delimiter=\\n' tox -e pylint-tests ; echo "$?" > pylint-tests-chg.exit) | tee pylint-report-tests-chg.log
                         # remove color escape coding
                         sed -ri 's/\\x1B\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g' pylint-report-tests.log
```

Comment:
@damon-atkins I think you need to change this to the `pylint-report-tests-chg.log` right?

```10:40:16 [lint salt chg] + sed -ri 's/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g' pylint-report-salt.log
10:40:16 [lint salt chg] sed: can't read pylint-report-salt.log: No such file or directory

https://jenkinsci.saltstack.com/job/pr-lint/job/PR-50670/4/console

Reply:
Good pickup. Fixed with phone.

Reply:
Nice!
It all looks good, approving.